Formula/vdirsyncer.rb

```diff
@@ -74,6 +74,45 @@ def install
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
+  def caveats
+    "The vdirsyncer launch script defaults to running every 60s with en_US.UTF-8 locale."
```

Comment:
Yes, as I wrote this plist and chose the params based on what I figured is fitting. And that I could share this with others. Not sure this is how it is supposed to be in homebrew though. I could ask upstream if they want to provide it.